benchmarking/run_synthetic_dataset.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# run_synthetic_dataset.sh
+
+python src/evaluator.py \
+--mode synthetic \
+--model-name "COE/Meta-Llama-3-8B-Instruct" \
+--results-dir "./data/results/llmperf" \
+--num-workers 1 \
+--timeout 600 \
+--num-input-tokens 1000 \
+--num-output-tokens 1000 \
+--num_requests 32 \
```

Comment:
Just for standardization, please change `--num_requests` to `--num-requests`

Reply:
✅